x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/data/X-VectorBlock.java.st

```diff
@@ -44,7 +43,7 @@ $endif$
     }
 
     @Override
-    public int getTotalValueCount() {
+    public int getPositionCount() {
```

Comment:
How about making the methods final?

Reply:
We should, but since this class is already final, we're good here.